tests/v2/validation/provisioning/config.go

```diff
@@ -31,6 +31,52 @@ const (
 	VsphereProviderName   ProviderName = "vsphere"
 )
 
+var AllRolesNode = machinepools.NodeRoles{
+	Etcd:         true,
+	ControlPlane: true,
+	Worker:       true,
+}
+
+var EtcdControlPlaneNode = machinepools.NodeRoles{
```

Comment:
Instead of having all these variations, would it make more sense just to have Etcd, ControlPlane, and Worker, and then in the tests define them in a list a such? Where use quantity like in the config file?

Reply:
sure, I can move this part to in the test files themselves.